File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/IWouldSubmit.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class IWouldSubmit : StoryChoice
	{
		internal IWouldSubmit() : base(StoryChoice.GetLocalizedString("ch5IWouldSubmit"), 5062)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5AnsweredCoach = PhysicalEducationAnswer.Submit;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/IWouldSubmit.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.PhysicalEducationAnswer;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class IWouldSubmit extends StoryChoice {
    public IWouldSubmit() {
        super(R.string.ch5IWouldSubmit, 5062);
    }

    @Override
    public void select()
    {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5AnsweredCoach(PhysicalEducationAnswer.Submit);
        super.select();
    }
}
